cdap-docs/build.sh

```diff
@@ -20,283 +20,210 @@
 # Builds each of these individually, and then packages them into a single zip file for distribution.
 # _common directory holds common files and scripts.
 
-# Optional Parameters (passed via env variable or exported in shell):
-# BELL: Set it to for the bell function to make a sound when called
-# COLOR_LOGS: Set it for color output by Sphinx and these scripts
-# NO_JAVADOCS: Set it to not build Javadocs, no matter which actions are used (for testing)
+# Optional Parameters (passed via environment variable or exported in shell):
+# BELL: Set for the bell function to make a sound when called
+# COLOR_LOGS: Set for color output by Sphinx and these scripts
+# USE_LOCAL: Set to use local copies of source, rather than downloading referenced files
+# NO_JAVADOCS: Set to not build Javadocs, no matter which actions are used (for testing)
 
 source ./vars
 source _common/common-build.sh
```

Comment:
This requires that this script is executed as `./build.sh` and it'll fail if invoked any other way. `cd $(cd $(dirname ${BASH_SOURCE[0]}); pwd -P)` will ensure your working directory is the canonical location for the script, no matter how it's called (unless it's called from a symlink outside this location, but we're not concerned about that use case)...